app/assets/stylesheets/generic/sidebar.scss

```diff
@@ -39,6 +39,42 @@
   display: inline-block !important;
 }
 
+.collapse-nav {
+  position: relative;
+  top: 50px;
+  width: 230px;
+  text-align: right;
+  padding-right: 21px;
+}
+
+.page-with-sidebar.collapsed {
+
+  .collapse-nav {
+    width: 53px;
+  }
+
+  padding-left: 50px;
+
+  .sidebar-wrapper {
+    width: 52px;
+    overflow-x: hidden;
+
+    .nav-sidebar {
+      width: 52px;
+    }
+    
+    .nav-sidebar li a > span {
+      display: none;
+    }
+  }
+}
+
+@media (max-width: 1198px) {
+  .collapse-nav {
+    display: none;
+  }
+}
+
```

Comment:
Navigation sidebar described in this css file - https://github.com/gitlabhq/gitlabhq/blob/master/app/assets/stylesheets/sections/nav_sidebar.scss